config/base/300-operator_v1alpha1_config_crd.yaml

```diff
@@ -67,6 +67,24 @@ spec:
               targetNamespace:
                 description: namespace where tekton components will be installed
                 type: string
+              pruner:
+                description: this enables to prune pipelinerun/taskrun
+                properties:
+                  namespace:
```

Comment:
Is it the namespace that will be cleaned ? if yes then you can only prune on one namespace or we have a way to prune on multiple ?

Reply:
with the current approach, that is true. 
How about taking multiple namespaces and creating a cron for each of it.?